extensions/emmet/extension.toml

```
@@ -1,15 +1,15 @@
 id = "emmet"
 name = "Emmet"
 description = "Emmet support"
-version = "0.0.4"
+version = "0.0.5"
```

```suggestion
version = "0.0.4"
```

We'll bump the version separately when we're ready to release a new version.